Src/WitsmlExplorer.Api/HttpHandlers/AuthorizeHandler.cs

```diff
@@ -35,5 +40,23 @@ public static IResult Deauthorize(IConfiguration configuration, HttpContext http
 
             return TypedResults.Ok();
         }
+
+        public static string GenerateToken(IConfiguration configuration, HttpRequest request, [FromServices] ICredentialsService credentialsService)
+        {
+            EssentialHeaders eh = new(request);
+            string sub = credentialsService.GetClaimFromToken(eh.GetBearerToken(), "sub");
+
+            SymmetricSecurityKey securityKey = new(Encoding.UTF8.GetBytes(configuration[ConfigConstants.NotificationsKey]));
+            SigningCredentials credentials = new(securityKey, SecurityAlgorithms.HmacSha256);
+            Claim[] claims = new[]
+            {
+                new Claim(ClaimTypes.NameIdentifier, sub),
```

Comment:
If you agree, we can change this to
```cs
new Claim("sub", sub),
```
to reduce token size. I believe it will behave identical